### DISCUSSION TITLE
Carbon Monoxide Detector

### DISCUSSION THREAD

**Discussion by iklein99** [iklein99/homebridge-smartthings#77] [General]
Anyone out there that can test the one of these for me?  I wrote the capability but can't test without a device.

**Comment by steinauf85** [comment]
i have CO detectors and can help test if needed.

**Comment by iklein99** [comment]
Give it a shot.  Already supported. Let me know how it goes. Sent from my iPhoneOn Dec 1, 2022, at 11:24 AM, steinauf85 ***@***.***> wrote:﻿
i have CO detectors and can help test if needed.
